docs/sage/10.x/installation.md

```diff
@@ -76,3 +76,26 @@ To install the latest development version of Sage, add `dev-main` to the end of
 ```sh
 $ composer create-project roots/sage your-theme-name dev-main
 ```
+
+## Build assets
+
+You must build theme assets in order to access your site. Failing to build the assets will result in the error:
+
+```
+The manifest [/path/to/sage/public/manifest.json] cannot be found.
```

Comment:
Is this the full error? We don't we include more details such as:

* telling them to run `bud build` 
* linking to asset docs

Reply:
yeah the error message isn't the greatest. need to bring that up on acorn

Reply:
If its from Acorn, then it should have a better generic message like telling users to run an npm/yard build command.

Can Sage customize messages like these from Acorn, then it could be made Sage specific? cc @QWp6t

Reply:
yeah, we can detect sage :+1: created https://github.com/roots/acorn/issues/204